exercises/practice/atbash-cipher/.meta/example.cpp

```diff
@@ -10,7 +10,7 @@ const char ALPHA_END = 'z';
 
 char atbash_transform(char c)
 {
-    return std::isalpha(c) ? ALPHA_END - std::tolower(c) + ALPHA_START : c;
+    return static_cast<char>(std::isalpha(c)) ? ALPHA_END - static_cast<char>(std::tolower(c)) + ALPHA_START : c;
```

Comment:
Atbash-cipher is failing on older versions. I think the first static cast is somehow causing it to always return false.

Maybe it needs to be `static_case<bool>(std::isalpha(static_cast<unsigned char>(c)))`?

See cppreference on isalpha: https://en.cppreference.com/w/cpp/string/byte/isalpha

Reply:
To clarify - wrapped the line with a static_cast<char> in order to fix the issue.